Finish make offer flows

adamsilver opened the pull request:
- Make an offer
- Make an offer to different course
- Make an offer to a different location
- Make an offer to a different training provider

To check/to do:

- all content
- what details we play back on every page considering consistency

laurahermionetennant92 commented:
@adamsilver Have added in some 'Candidate preferences' to match 'Your existing offer' in the change offer flow. 
When I get to this page I expect to see the updated course, location and conditions (maybe next door to the Candidate preferences0?
![Screenshot 2020-03-16 at 17 09 13](https://user-images.githubusercontent.com/38726669/76783041-15237280-67a9-11ea-859a-58cdcadf1977.png)

laurahermionetennant92 commented:
Working through make offer change location flow. I'm now wondering if the updated location should be displayed here before we get to the conditions (otherwise confusing?). Possibly with a banner saying 'You have successfully changed the location for this candidate's application'
![Screenshot 2020-03-16 at 17 14 14](https://user-images.githubusercontent.com/38726669/76783370-ab579880-67a9-11ea-8ea5-eed21033795c.png)

adamsilver commented:
Okay I've worked on this last night and this morning.

- Made all content consist across all 5 journeys
- Removed playback from most pages as there's not been a need demonstrated for it
- Reinstated caption to provide minimal context which hasn't been shown to be a problem in live
- Made existing flows match live in capability i.e. removed/hidden recommendations
- as per @laurahermionetennant92 work from change offer, the first screen of every flow that is a ‘change’ says ‘Select alternative [the thing]
- Playback of conditions is just ‘conditions’ and not ‘standard’ and ‘further’
- Use a standard check answers pattern so now user can change their answers on each flow.
